### ISSUE TITLE
"Font not set" error when logging in

### ISSUE THREAD

**Issue by olepg** [Aviana/LunaUnitFrames#948]
I get a LUA error everytime I log onto my death knight. Even though I use the same profile for many toons, this only happens on my death knight. If I select a different profile, then back the issue resolves it self, but returns when I reload the UI.
This is the error:

```lua
Message: ...bs\oUF_Plugins\oUF_TagsWithHeal\oUF_TagsWithHeal.lua:1533: <unnamed>:SetFormattedText(): Font not set
Time: Tue Sep 20 15:37:46 2022
Count: 1
Stack: ...bs\oUF_Plugins\oUF_TagsWithHeal\oUF_TagsWithHeal.lua:1533: <unnamed>:SetFormattedText(): Font not set
[string "=[C]"]: ?
[string "=[C]"]: in function `SetFormattedText'
[string "@Interface\AddOns\LunaUnitFrames\libs\oUF_Plugins\oUF_TagsWithHeal\oUF_TagsWithHeal.lua"]:1533: in function `UpdateTag'
[string "@Interface\AddOns\LunaUnitFrames\libs\oUF_Plugins\oUF_TagsWithHeal\oUF_TagsWithHeal.lua"]:1410: in function `func'
[string "@Interface\AddOns\LunaUnitFrames\libs\oUF\ouf.lua"]:215: in function <Interface\AddOns\LunaUnitFrames\libs\oUF\ouf.lua:198>
[string "=(tail call)"]: ?
[string "=(tail call)"]: ?

Locals:
```

```lua
Message: ...bs\oUF_Plugins\oUF_TagsWithHeal\oUF_TagsWithHeal.lua:1533: <unnamed>:SetFormattedText(): Font not set
Time: Tue Sep 20 15:37:51 2022
Count: 5
Stack: ...bs\oUF_Plugins\oUF_TagsWithHeal\oUF_TagsWithHeal.lua:1533: <unnamed>:SetFormattedText(): Font not set
[string "=[C]"]: ?
[string "=[C]"]: in function `SetFormattedText'
[string "@Interface\AddOns\LunaUnitFrames\libs\oUF_Plugins\oUF_TagsWithHeal\oUF_TagsWithHeal.lua"]:1533: in function `UpdateTag'
[string "@Interface\AddOns\LunaUnitFrames\libs\oUF_Plugins\oUF_TagsWithHeal\oUF_TagsWithHeal.lua"]:1366: in function <...bs\oUF_Plugins\oUF_TagsWithHeal\oUF_TagsWithHeal.lua:1361>

Locals:
```

**Comment by Aviana** [comment]
Did you change any of the Tags options or are they the default?
Do you use another font than the default one?
Since when do you have this problem?

**Comment by olepg** [comment]
Did you change any of the Tags options or are they the default?
Yes, I've changed them.

```
Health bar left: [name]
Health bar right: [ssmarthealth] [perhp]%

Power bar left: [levelcolor][level][shortclassification] [classcolor][smartclass]
Power bar right: [pp]/[maxpp] [perpp]%
```

Do you use another font than the default one?
Yes. I've set the fonts to Futura.

Since when do you have this problem?
This problem started when I created the DK and logged into it.

**Comment by olepg** [comment]
Did you change any of the Tags options or are they the default?
Yes, I've changed them.

```
Health bar left: [name]
Health bar right: [ssmarthealth] [perhp]%

Power bar left: [levelcolor][level][shortclassification] [classcolor][smartclass]
Power bar right: [pp]/[maxpp] [perpp]%
```

Do you use another font than the default one?
Yes. I've set the fonts to Futura.

Since when do you have this problem?
This problem started when I created the DK and logged into it.

**Comment by Aviana** [comment]
Should be fixed in 4.337
